.devcontainer/docker-compose.yml

```diff
@@ -19,6 +19,8 @@ services:
       CIRCUITVERSE_USE_SOLR: "false"
       DOCKER_ENVIRONMENT: "true"
       DEV_CONTAINER: "true"
+    volumes:
+    - ..:/workspaces/CircuitVerse:cached
```

Comment:
`:cached` probably has been deprecated after introducing of FUSE file sharing in docker desktop.

Ref: https://github.com/docker/for-mac/issues/5402 

Can you try out to remove `:cached` flag, if it still works ?

Reply:
@tanmoysrt I force-pushed the ammend commit to this branch. Do i need to create a New PR or are the changes visible within this PR ?